### PR TITLE
COMMERCE-5042 reorganize language keys + provide main dispatch configuration independent of talend configuration

### PR DESCRIPTION
**Pull request by igorbeslic**

### CODE REVIEW

**Comment by igorbeslic** [comment]
ci:forward

**Comment by igorbeslic** [comment]
@matijapetanjek

File: modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+an-error-occurred-while-processing-the-scheduled-task=An error occurred while processing the scheduled task.
+are-you-sure-you-want-to-delete-the-selected-logs=Are you sure you want to delete the selected logs?
+are-you-sure-you-want-to-delete-the-selected-trigger=Are you sure you want to delete selected trigger?
+cron-expression=Cron Expression
+dispatch=Dispatch
+dispatch-configuration-name=Dispatch Configuration Name
+dispatch-trigger=Dispatch Trigger
+edit-trigger=Edit Trigger
+execute-once=Execute Once
+last-fire-time=Last Run Time
```
Comment (dsitu) [inline modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties:14]: @igorbeslic looks like some of the existing keys carried over don't have matching keys and values, I think we should fix them.
`last-run-time=Last Run Time`

File: modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+last-update=Last Update
+launch-type=Launch Type
+logs=Logs
+never-end=Never End
+next-fire-date=Next Run Date
```
Comment (dsitu) [inline modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties:19]: `next-run-date=Next Run Date`

File: modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+not-yet-executed=Not Yet Executed
+not-yet-scheduled=Not Scheduled Yet
```
Comment (dsitu) [inline modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties:21]: `not-scheduled-yet=Not Scheduled Yet`

File: modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+overlap-allowed=Overlap Allowed
+process=Process
+processes-admin=Processes
```
Comment (dsitu) [inline modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties:24]: `processes=Processes`

File: modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+run-now=Run Now
+runtime=Runtime
+task-already-running=Task is already running.
```
Comment (dsitu) [inline modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties:27]: `task-is-already-running=Task is already running.`

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by igorbeslic** [comment]
@dsitu thank you, I'll do it in next batch

**Pull request closed without merging**